styles.css

```diff
@@ -538,4 +528,264 @@ textarea::placeholder {
   .see-project-btn {
     width: 50%;
   }
+
+  .js-img img,
+  .html-img img,
+  .css-img img {
+    margin-top: 1rem;
+    width: 90%;
+    border-radius: 0.5rem;
+  }
+
+  .framework-list,
+  .skill-list {
+    display: flex;
+    justify-content: space-between;
+    padding: 1rem;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .header {
+    padding-left: 155px;
+  }
+
+  .nav {
+    margin-right: 154px;
+  }
+
+  a.logo {
+    font-size: 28px;
+  }
+
+  .logo-section {
+    padding-top: 24px;
+  }
+
+  .bars {
+    display: none;
+  }
+
+  .nav-links {
+    display: flex;
+    visibility: visible;
+    padding-top: 25px;
+    padding-bottom: 25px;
+  }
+
+  .nav-links img {
+    display: none;
+  }
+
+  .nav-links a {
+    margin-right: 25px;
+    font-size: 22px;
+  }
+
+  .hero-section {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 700px;
+    margin: 50px auto;
+    padding-left: 400px;
+    border-bottom-left-radius: 6rem;
+    background-image: url("images/Headerdesktopbg.svg");
+    background-repeat: no-repeat;
+    background-size: 100%;
+  }
+
+  .full-description .about-me {
+    width: 45%;
+    padding-top: 12px;
+    padding-bottom: 12px;
+  }
+
+  .about-rose .full-description .about-me {
+    width: 60%;
+    padding-top: 18px;
+    padding-bottom: 18px;
+    font-size: 14px;
+  }
+
+  .card {
+    width: 1156px;
+    height: 30rem;
+    flex-direction: row;
+    padding: 16px 32px 16px 32px;
+    margin: 2rem auto;
+    border-radius: 24px;
+  }
+
+  .card .first-column {
+    width: 50%;
+  }
+
+  .card .second-column {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    padding-left: 20px;
+  }
+
+  a.see-project-btn {
+    width: 30%;
+    margin-top: 2rem;
+    margin-left: 0;
+  }
+
+  .project-title {
+    margin-top: 28px;
+    font-size: 2.5rem;
+    font-weight: 900;
+  }
+
+  .project-description p {
+    width: 58%;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+  }
+
+  .project-image img {
+    height: 100%;
+    margin: 0 auto;
+  }
+
+  .art-project-img,
+  .last-project-img {
+    order: 2;
+  }
+
+  .art-project-description,
+  .last-project-description {
+    order: 1;
+    padding-left: 40px;
+  }
+
+  .about-rose {
+    border-top-right-radius: 0;
+    margin-top: 0;
+    padding-left: 200px;
+    width: 50%;
+    height: 650px;
```

Comment:
- [ ] The elements in your `about me` section appear to be positioned wrongly. Some elements appear to be overflowing with other elements. Kindly position them correctly to match the original design.

![image](https://user-images.githubusercontent.com/98453979/220686114-e6558ec0-849e-43ae-b592-6fe23ac908de.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/220686393-7c9758d4-e038-4537-a720-480ec3cf0457.png)

- [ ] On your mobile version,  I would like to encourage you to push the headline section down so that there is enough spacing from the toolbar like in the original design

![image](https://user-images.githubusercontent.com/98453979/220687397-53362719-c87e-4724-98e3-140f2005cc31.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/220687618-1e2ddabe-6d26-472b-a3fa-0de570d62d9d.png)